Add Credix1 adapter for Unit0 network

iusserhat opened the pull request:
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):


##### Twitter Link:


##### List of audit links if any:


##### Website Link:


##### Logo (High resolution, will be shown with rounded borders):


##### Current TVL:


##### Treasury Addresses (if the protocol has treasury)


##### Chain:


##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)


##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)


##### Short Description (to be shown on DefiLlama):


##### Token address and ticker if any:


##### Category (full list at https://defillama.com/categories) *Please choose only one:


##### Oracle Provider(s): Specify the oracle(s) used (e.g., Chainlink, Band, API3, TWAP, etc.):
##### Implementation Details: Briefly describe how the oracle is integrated into your project:
##### Documentation/Proof: Provide links to documentation or any other resources that verify the oracle's usage:

##### forkedFrom (Does your project originate from another project):


##### methodology (what is being counted as tvl, how is tvl being calculated):


##### Github org/user (Optional, if your code is open source, we can track activity):

llamabutler commented:
The adapter at projects/credix1 exports TVL: 
        
 
 
    unit0                     0
    
    total                    0

waynebruce0x commented on projects/credix1/index.js:
```diff
@@ -0,0 +1,29 @@
+  try {
+    // Token bakiyesini kontrol etmeye çalışıyoruz
+    return sumTokens2({ api, owners: [p2p], tokens: [token] });
+  } catch (error) {
```
please remove any try/catch blocks, this will prevent our error monitoring

llamabutler commented:
The adapter at projects/credix1 exports TVL: 
        
 
 
    unit0                     0
    
    total                    0

llamabutler commented:
The adapter at projects/credix1 exports TVL: 
        
 
 
    unit0                     0
    
    total                    0

waynebruce0x commented:
what is 0xB13Ae312BC7c45d7120bf8bf317d5f55C36b19dF? Who mints it?

iusserhat commented:
> what is 0xB13Ae312BC7c45d7120bf8bf317d5f55C36b19dF? Who mints it?

We have not integrated Oracle in our project yet, we are using the token we created for collateral in the credit system, we will integrate Oracle in the later stages of our roadmap.

iusserhat commented:
> > what is 0xB13Ae312BC7c45d7120bf8bf317d5f55C36b19dF? Who mints it?
> 
> We have not integrated Oracle in our project yet, we are using the token we created for collateral in the credit system, we will integrate Oracle in the later stages of our roadmap.

I just realized that we had created our own token on the testnet. Now that we've moved to the mainnet, we actually don’t need our own token. I’ve changed the USDT address to the collateral token address.

iusserhat commented:
> > > what is 0xB13Ae312BC7c45d7120bf8bf317d5f55C36b19dF? Who mints it?
> > 
> > 
> > We have not integrated Oracle in our project yet, we are using the token we created for collateral in the credit system, we will integrate Oracle in the later stages of our roadmap.
> 
> I just realized that we had created our own token on the testnet. Now that we've moved to the mainnet, we actually don’t need our own token. I’ve changed the USDT address to the collateral token address.

We’ve also updated our smart contracts to work with USDT.

llamabutler commented:
The adapter at projects/credix1 exports TVL: 
        
 
 
    unit0                     0
    
    total                    0

Define101 commented:
@iusserhat fill out the form or at least website and twitter link

iusserhat commented:
> @iusserhat fill out the form or at least website and twitter link

I added
